packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -73,6 +74,48 @@ import 'ticker_provider.dart';
 /// }
 /// ```
 /// {@end-tool}
+///
+/// Instead of switching to [NestedScrollView], the position where the glow is
+/// drawn can also be manipulated by making use of a
```

Comment:
```suggestion
/// The position where the glow is drawn in a [CustomSCrollView] can be 
/// manipulated by making use of a
```

Reply:
I dropped the whole paragraph, since it was redundant after moving the example before the `NestedScrollView` example.